<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light Tasks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/icons/lighttasks.png</normaloff>:/icons/lighttasks.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="taskListScrollArea">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="taskListWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>222</width>
         <height>262</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="taskListLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>269</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mainOperationButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Add new task</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>mainOperationButton</tabstop>
  <tabstop>taskListScrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
